sklearn/decomposition/_fastica.py

```diff
@@ -9,14 +9,14 @@
 #          Bertrand Thirion, Alexandre Gramfort, Denis A. Engemann
 # License: BSD 3 clause
 
+from email.mime import base
```

Comment:
This import is unused and must be removed.

(I guess this might have been imported via IDE code completion?)

```suggestion
```